Code/Legacy/CryCommon/ISystem.h

```diff
@@ -9,9 +9,9 @@
 #pragma once
 
 #ifdef CRYSYSTEM_EXPORTS
-#define CRYSYSTEM_API DLL_EXPORT
+#define CRYSYSTEM_API AZ_DLL_EXPORT
```

Comment:
Repeat: I believe AzCore/PlatformDef.h needs to be included to have AZ_DLL_EXPORT/AZ_DLL_IMPORT defined.